diff --git a/libfaac/Makefile.am b/libfaac/Makefile.am
index 6211c82..044b372 100644
--- a/libfaac/Makefile.am
+++ b/libfaac/Makefile.am
@@ -6,18 +6,14 @@ if CPUSSE
 common_CFLAGS += -msse2
 endif
 
-if USE_DRM
-lib_LTLIBRARIES = libfaac_drm.la
+lib_LTLIBRARIES = libfaac_drm.la libfaac.la
 libfaac_drm_la_SOURCES = ${common_SOURCES} ${common_INCLUDES} kiss_fft/kiss_fftr.c kiss_fft/kiss_fft.c kiss_fft/kiss_fft.h
 EXTRA_libfaac_drm_la_SOURCES = kiss_fft/_kiss_fft_guts.h kiss_fft/kiss_fftr.h kiss_fft/README kiss_fft/README.kiss_fft
 libfaac_drm_la_LIBADD = ${common_LIBADD}
 libfaac_drm_la_CFLAGS = ${common_CFLAGS} -DDRM
-else
-lib_LTLIBRARIES = libfaac.la
 libfaac_la_SOURCES = ${common_SOURCES} ${common_INCLUDES}
 libfaac_la_LIBADD = ${common_LIBADD}
 libfaac_la_CFLAGS = ${common_CFLAGS}
-endif
 
 libfaac_la_LDFLAGS = -no-undefined
 if MINGW
